geotrek/core/tests/test_commands.py

```diff
@@ -229,3 +230,353 @@ def test_load_paths_within_spatial_extent_no_srid_geom(self):
         value = Path.objects.first()
         self.assertEqual(value.name, 'lulu')
         self.assertEqual(value.structure, self.structure)
+
+
+@skipIf(not settings.TREKKING_TOPOLOGY_ENABLED, 'Test with dynamic segmentation only')
+class ReorderTopologiesPathAggregationTest(TestCase):
+    def setUp(self):
+        """
+        ⠳               ⠞
+          ⠳           ⠞
+            ⠳       ⠞
+              ⠳   ⠞
+                ⠿
+              ⠞   ⠳
+            ⠞       ⠳
+        1 ⠞           ⠳ 2
+        ⠞               ⠳
+        """
+        self.path_1 = PathFactory.create(geom=LineString(Point(700000, 6600000), Point(700100, 6600100),
```

Comment:
Only for your information, since python3.6, you can add '_' in numeric litterals to make them more readable (the '_' characters are ignored when parsing the numeric litterals).
For instance : `LineString(Point(700_000, 6_600_000), Point(700_100, 6_600_100))`